### PR TITLE
hydra-send-stats: fix imports

### PR DESCRIPTION
**Pull request by Mic92**
fixes https://github.com/NixOS/hydra/issues/894

### CODE REVIEW

**Comment by Mic92** [comment]
Tested in nix-community: https://github.com/nix-community/infra/commit/a9295a7298a46bde2d1ef89a983e1e47da21e9bf

File: src/script/hydra-send-stats [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
+use Hydra::Helper::Nix;
 use Net::Statsd;
 use File::Slurp;
 use JSON;
 
 STDERR->autoflush(1);
 binmode STDERR, ":encoding(utf8)";
 
-my $config = getHydraConfig();
+my $config = Hydra::Helper::Nix::getHydraConfig();
```
Comment (grahamc) [inline src/script/hydra-send-stats:13]: `use` brings it in to scope by default, so this part isn't necessary. I'll fix this up in a push I'm about to make adding some tests. Thanks!

Comment (grahamc) [reply src/script/hydra-send-stats:13]: (I know I made this mistake one line lower. I know more perl than I did then :'))

**Review by grahamc — APPROVED** [review]
I pushed to your branch adding a test, making sure it failed, and that your commit fixed it. I also ran the test without "--once" to make sure it took "forever". It'd be nice to have a more thorough test, but I'm happy enough with "does it run without exiting badly?" for now.

**Comment by Mic92** [comment]
I deployed this patch to nix-community.

**Pull request merged**